test/e2e/cypress/support/commands.js

```diff
@@ -92,6 +92,18 @@ Cypress.Commands.add('longRecoveryLifespan', ({} = {}) => {
     return config
   })
 })
+Cypress.Commands.add('dontLoginUserAfterRegistration', ({} = {}) => {
+  updateConfigFile((config) => {
+    delete config.selfservice.flows.registration.after['password']
```

Comment:
Similar to my comment below this should probably have a filter. Also, we want the inverse (so add session back in)